### ISSUE TITLE
plone namespace

### ISSUE THREAD

**Issue by jensens** [nazrulworld/plone.oidc#3]
@nazrulworld As this is in the ``plone.`` namespace it would be great if you transfer it to the Plone Foundation.
Otherwise the `pas.plugins.` or `collective.` namespaces would been a better choice.

**Comment by nazrulworld** [comment]
Yes, that was the original plan to transfer to plone. Unfortunately, I cannot manage time for this project so this product is not nearly close to useable.
That's why this product still in here.

**Comment by jensens** [comment]
OK, I am just evaluating OIDC solutions for Plone for a customer. So I have the Authomatic based approach which in theory (I need to test it) works and found yours here. But pas.plugins.authomatic needs upgrade to Python 3 yours is unfinished, also Authomatic is not very actively developed. Difficult.  :D

**Comment by nazrulworld** [comment]
I fully understand :)
Correct me if I wrong,  `pas.plugins.automatic is focusing on mostly service providers(consumers of IdP), not as identity provider itself.
Here my idea was as an Identity Provider implementation, especially the implementation of https://openid.net/wg/heart/  
If your use case is that you will need to consume third-party IdP, then in my opinion `pas.plugins.authomatic` would be a better option.
But if your requirements as Identity Provider, then I have an idea, (if you have time ) I can transfer this project to plone immediately and give you admin role, I am open to refactoring everything, for example, license. 
I could manage some time to contribute as well.

**Comment by jensens** [comment]
Thank for pointing this out. I need a consumer for now. So I'll better go for pas.plugins.authomatic. In the other case I would have loved to contribute here!
